core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.component.ts

```diff
@@ -204,6 +206,15 @@ export class SkillConceptCardEditorComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.directiveSubscriptions.add(
+      this.resizeSubscription =
```

Comment:
I don't think we need another property for the resize subscription, since we're adding all the subscriptions to the `directiveSubscriptions` property. `directiveSubscriptions` is enough to keep track of this subscription, so you can get rid of this line (and the `resizeSubscription` property).

Reply:
@nithinrdy done with the changes. PTAL!